server/sonar-server/src/main/java/org/sonar/server/rule/ws/SearchAction.java

```diff
@@ -145,7 +145,8 @@ public void define(WebService.NewController controller) {
         "<li>\"debtRemFn\" becomes \"remFn\"</li>" +
         "<li>\"effortToFixDescription\" becomes \"gapDescription\"</li>" +
         "<li>\"debtOverloaded\" becomes \"remFnOverloaded\"</li>" +
-        "</ul>")
+        "</ul>" +
+        "In 7.1, the field 'scope' has been added.")
```

Comment:
Please remove this from the description and add it to the changelog of the action (see next comment)